spyder/plugins/explorer/widgets/explorer.py

```diff
@@ -712,7 +712,8 @@ def update_actions(self):
 
         # Make actions visible conditionally
         self.move_action.setVisible(
-            all([fixpath(osp.dirname(fname)) == basedir for fname in fnames]))
+            all([fixpath(osp.dirname(fname)) == basedir for fname in fnames])
+            and only_files)
```

Comment:
```suggestion
            all(
                [fixpath(osp.dirname(fname)) == basedir for fname in fnames])
                and only_files
            )
```

Code style improvement.